chartos/tests/test_data.py

```diff
@@ -1,20 +1,19 @@
-import pyproj
-from shapely.geometry import shape
-from shapely.ops import transform
-
-
-campus_sncf_gps = shape({
-    "type": "Point",
-    "coordinates": [
-        2.3538637161254883,
-        48.9210857797314
-    ]
-})
+campus_sncf_bbox = [[2.3535, 48.921], [2.3568, 48.922]]
 
 ref_tiles = {
     (132786, 90113, 18),
+    (132786, 90112, 18),
+    (132788, 90112, 18),
+    (132788, 90113, 18),
+    (132785, 90112, 18),
+    (132785, 90113, 18),
+    (132787, 90112, 18),
+    (132787, 90113, 18),
+    (66394, 45056, 17),
     (66393, 45056, 17),
+    (66392, 45056, 17),
     (33196, 22528, 16),
+    (33197, 22528, 16),
```

Comment:
why did the test result change?

Reply:
Instead of an arbitrary geometry, the function that calculates the tiles to invalidate takes a bbox as a parameter.
I replaced `campus_sncf_gps` by `campus_sncf_bbox`. That's why the result has more tiles.